__tests__/convertors.test.js

```diff
@@ -0,0 +1,65 @@
+import { defaultBreakpoints } from '../src/index';
+import { pxToEm, pxToRem } from '../src/convertors';
+
+export const itBreakPoints = {
+  huge: '1440',
+  large: '1170',
+  medium: '768',
+  small: '450',
+};
+
+describe(`pxToEm()`, () => {
+  it('pxToEm to equal defaultBreakpoints converted to em', () => {
+    expect(pxToEm(defaultBreakpoints, 16)).toEqual({
```

Comment:
suggestion: In this file, all those `.toEqual` could be replaced to `.toMatchInlineSnapshot`. This would bring the same value and capability to change/refactor as doing manually.

Reply:
I once was a fan of snapshot testing, but wouldn't recommend it in this case - especially when thinking about CI/CD. `.toMatch[Inline]Snapshot()` gets overwritten with each deployment, that's why I rather equality tests than them.